src/DotNetWorker.Core/Http/HttpResponseDataExtensions.cs

```diff
@@ -120,7 +132,7 @@ public static ValueTask WriteAsJsonAsync<T>(this HttpResponseData response, T in
         }
 
         /// <summary>
-        /// Write the specified value as JSON to the response body using the default <see cref="ObjectSerializer"/> configured for this worker.
+        /// Asynchronously write the specified value as JSON to the response body using the default <see cref="ObjectSerializer"/> configured for this worker.
```

Comment:
I think this is still grammatically correct, but if you want to match the other summaries this is missing an 's' (_writes_ not write).

Reply:
Good eye 🔍. Fixed.